_roadmap/01_plans.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: single
+sidebar:
+  nav: plans.md
+title: Plans for upcoming challenges
+excerpt: Plans for upcoming SAMPL challenges and phases
+permalink: /roadmap/plans/
+---
+
+## SAMPL overview, SAMPL7-10
+
+As of SAMPL7-10, our NIH funding allows us to be somewhat more strategic in planning challenges, so we plan a series of challenges encompassing:
+- Physical property prediction (e.g. pKa, logP, logD, etc.)
+- Host-guest binding
+- Protein-ligand binding
+
+SAMPL is now broken into phases, each running roughly a year, with each phase encompassing several component challenges. [Workshop plans](https://samplchallenges.github.io/current/workshops/) are detailed separately.
+
+The Gibb and Isaacs groups, focusing on host-guest binding, plan to collect regular data on host-guest binding, and protein-ligand binding assays are coming online in the Chodera lab. Physical property data, however, relies on planned industry internships and other partnerships.
+
+## SAMPL7 plans (present through late 2019)
+
+SAMPL7 is planned to include:
+- "SAMPL6 part 2" logP challenge (closed March, 2019)
```

Comment:
I think we shouldn't list SAMPL6 part 2 logP challenge as a part of SAMPL7 here. It would be better to remove this line.

Reply:
OK, thanks.